### PR TITLE
release: 2.51.2

### PR DESCRIPTION
**Pull request by mvo5**
Release 2.51.2 changelog to master

### CODE REVIEW

File: packaging/debian-sid/changelog [hunk 1]
```diff
@@ -1,3 +1,22 @@
+snapd (2.51.2-1) unstable; urgency=medium
+
+  * New upstream release, LP: #1929842
+    - snapstate: remove temporary snap file for local revisions early
+    - interface: allows reading sd cards internal info from block-
+      devices interface
+    - o/ifacestate: do not visit same halt tasks in waitChainSearch to
+      avoid cycles
+    - corecfg: allow using `# snapd-edit: no` header to disable pi-
+      config# snapd-edit: no
```
Comment (pedronis) [inline packaging/debian-sid/changelog:10]: this seems to have too much at the end

File: packaging/debian-sid/changelog [hunk 2]
```diff
@@ -1,3 +1,22 @@
+snapd (2.51.2-1) unstable; urgency=medium
+
+  * New upstream release, LP: #1929842
+    - snapstate: remove temporary snap file for local revisions early
+    - interface: allows reading sd cards internal info from block-
+      devices interface
+    - o/ifacestate: do not visit same halt tasks in waitChainSearch to
+      avoid cycles
```
Comment (pedronis) [inline packaging/debian-sid/changelog:8]: oh well, I thought I had updated the summary of that one, but apparently didn't, it's a bit misleading

**Review by pedronis — COMMENTED** [review]
there's some spurious stuff in one of the entries

File: packaging/debian-sid/changelog (hunk 2 above)
Comment (mvo5) [reply packaging/debian-sid/changelog:8]: What should it say instead? I can fix the changelog trivially

Comment (pedronis) [reply packaging/debian-sid/changelog:8]: to avoid slow convergence (or unlikely cycles)

**Review by pedronis — APPROVED** [review]
thanks

**Pull request merged**